website/routes.py

```diff
@@ -70,6 +71,19 @@ def set_status_message(user):
             extra={}
         )
 
+def anonymize_ip(request):
+    remote_addr = request.remote_addr
+    if remote_addr:
+        try:
+            ip = ipaddress.ip_address(unicode(remote_addr))
```

Comment:
For some reason I had to pass in unicode to get this to work.

Reply:
my guess is this is a py3 lib